exercises/armstrong-numbers/src/test/java/ArmstrongNumbersTest.java

```diff
@@ -0,0 +1,87 @@
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class ArmstrongNumbersTest {
+
+    private ArmstrongNumbers armstrongNumbers;
+
+    @Before
+    public void setup() {
+        armstrongNumbers = new ArmstrongNumbers();
+    }
+
+    @Test
+    public void singleDigitsAreArmstrongNumbers() {
+        int input = 5;
+        boolean expected = true;
+
+        assertEquals(expected, armstrongNumbers.isArmstrongNumber(input));
```

Comment:
You can use `assertTrue` and `assertFalse` instead of `assertEquals` for these tests :)